holidayextras/ui-toolkit
# New lists and icons

#### What does this PR do? (please provide any background)
- Replaces the List Item component and replaces it with a more robust atomic set of lists.
- This includes Description Lists, Unordered Lists, Ordered Lists and Icon list.

#### What tests does this PR have?
- Tests for all JSX files.

#### How can this be tested?
- Run `npm run docs` and look at the documentation, you can edit the code on the docs page, to see how the lists work
- Feel free to ask me any questions or come see it working on my machine.

#### Screenshots / Screencast
<img width="854" alt="screen shot 2015-10-27 at 14 19 14" src="https://cloud.githubusercontent.com/assets/778942/10760701/c5c95480-7cb5-11e5-8a9a-10faad711c0c.png">
<img width="849" alt="screen shot 2015-10-27 at 14 19 30" src="https://cloud.githubusercontent.com/assets/778942/10760705/c8046a64-7cb5-11e5-8f06-beaf5eaed468.png">

#### What gif best describes how you feel about this work?
![poppy](https://media.giphy.com/media/nJj5COuQU2qOc/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
